.github/workflows/test.yml

```
@@ -8,12 +8,12 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: ['14', '16', '18', '20']
```

I think extremes we want to support 
```suggestion
        node-version: ['14', '20']
```